README.md

```diff
@@ -547,6 +547,23 @@ const uiSchema = {
 };
 ```
 
+Unlike other types, array's do not support the `required` property.  You can however instead specify the minimum number of items the user must select with the `minItems` property.
```

Comment:
No problem!  While it's true supports the required property, I suspect that it doesn't operate in the fashion most UI builders would expect, at the very least it's a potential source of confusion.  I'd try and update the documentation to be clear in that regard. :)

And yes to the tests as well! :)